QUEST.tsv

```diff
@@ -2707,56 +2707,56 @@ QUEST_20151016_002714	Only you...{nl}Only the Revelator can save this world.{nl}
+QUEST_20151016_002732	{memo X}Below, I believe there's someone destroying Nestospa's Monuments by disguising as Anastospa.{nl}Can you go and investigate?{nl}It might be dangerous, take this flare.{nl}Be careful.
 QUEST_20151016_002733	$Kedoran Merchant Alliance Notice
 QUEST_20151016_002734	$Recruiting new members for the Kedoran Merchant Alliance!
-QUEST_20151016_002735	
-QUEST_20151016_002736	
-QUEST_20151016_002737	
-QUEST_20151016_002738	
-QUEST_20151016_002739	
-QUEST_20151016_002740	
-QUEST_20151016_002741	
-QUEST_20151016_002742	
-QUEST_20151016_002743	
-QUEST_20151016_002744	
-QUEST_20151016_002745	
-QUEST_20151016_002746	
-QUEST_20151016_002747	
-QUEST_20151016_002748	
-QUEST_20151016_002749	
-QUEST_20151016_002750	
-QUEST_20151016_002751	
-QUEST_20151016_002752	
-QUEST_20151016_002753	
-QUEST_20151016_002754	
-QUEST_20151016_002755	
-QUEST_20151016_002756	
-QUEST_20151016_002757	
-QUEST_20151016_002758	
-QUEST_20151016_002759	
-QUEST_20151016_002760	
-QUEST_20151016_002761	
-QUEST_20151016_002762	
-QUEST_20151016_002763	
-QUEST_20151016_002764	
-QUEST_20151016_002765	
-QUEST_20151016_002766	
-QUEST_20151016_002767	
+QUEST_20151016_002735	Nestospa's Member
+QUEST_20151016_002736	A godess scripture can be be made by paper and binding of leather straps found in Genra Field.
+QUEST_20151016_002737	Thrown away Gedas' research materials.
+QUEST_20151016_002738	The flare went off. Fabius is in danger.
+QUEST_20151016_002739	Now. Let's see.
+QUEST_20151016_002740	Oh, I think I found it.{nl}You see that box over there? That's it.
+QUEST_20151016_002741	Why are there only seniors there?{nl}Darn.. Is this monocle a sham?{nl}
+QUEST_20151016_002742	Though, it's good that the senior knew about the revelation you've been looking for.{nl}Anyway, did you talk about me?{nl}You did well, you did well.{nl}
+QUEST_20151016_002743	Anyway, it seems that you need a certification to head down to another area.{nl}Can you leave a note so that I can get a certification later as well?{nl}
+QUEST_20151016_002744	There's something I need to find out.{nl}Let's go in our separate ways while keeping in contacts with each other.You understand?
+QUEST_20151016_002745	Huh? What is this..{nl}My strength.. My body.. It's crumbling?{nl}
+QUEST_20151016_002746	No! How could people like you ruin my last 600 years of effort?{nl}How could a humans! How dare you humans!!{nl}
+QUEST_20151016_002747	My faithful Mandara!{nl}Shred those ignorant Revelator!!{nl}
+QUEST_20151016_002748	Ruklys.. It won't go your way..{nl}Never.. Revelation..
+QUEST_20151016_002749	Saviors, who followed all the way here by having faith in the revelation.{nl}You're gradually getting closer to the truth.{nl}
+QUEST_20151016_002750	Saviors. What's a goddess to humans.{nl}Symbol of calamity, the root of everyone's sorrow..{nl}
+QUEST_20151016_002751	No.{nl} That's the image created by demon's crooked hearts.{nl}
```

Comment:
That's the image created by demon's crooked hearts

to

That's the image created by demons' crooked hearts